src/Microsoft.Azure.SignalR.Common/ServiceConnections/MultiEndpointServiceConnectionContainer.cs

```diff
@@ -305,7 +305,7 @@ private async Task RemoveHubServiceEndpointAsync(HubServiceEndpoint endpoint)
                     return;
                 }
 
-                _ = container.ConnectionContainer.OfflineAsync(false);
+                _ = container.ConnectionContainer.OfflineAsync(GracefulShutdownMode.Off);

```

Comment:
@JialinXin Any problem here?

Reply:
Looks good to me.

---
In reply to: [422863751](https://github.com/Azure/azure-signalr/pull/904#discussion_r422863751) [](ancestors = 422863751)